tests/template_tests/syntax_tests/test_basic.py

```diff
@@ -346,6 +346,61 @@ def test_ignores_strings_that_look_like_format_interpolation(self):
         output = self.engine.render_to_string("tpl-weird-percent")
         self.assertEqual(output, "% %s")
 
+    @setup({"template": "{{ class_var.in_template }} | {{ class_var.render_class }}"})
+    def test_access_class_not_initiated(self):
```

Comment:
This test title feels a little misleading, shall we instead use something like this?
```suggestion
    def test_subscriptable_class(self):
```